File: multiconverter.js
const prompt = require('prompt-sync')()
var start = prompt("Do you want to convert a value? yes/no ")
if (start == yes){
    var value = prompt("Which conversion? (euros/celsius/inches)")
        if(value == "euros"){
            console.log(exchange(euros))
        }else if(value == "celsius"){
            console.log(temp_conv(celsius))
        }else if(value== "inches"){
            console.log(inchTocm())
        }
}else{
    process.exit()
}

function exchange(euros){
    return euros + " euros are " + (euros*0.98).toString() + " CHF"
}

function temp_conv(celsius){
    msg_1 = " degrees Celsius are "
    msg_2 = " degrees Farenheit"
    result = celsius * 9/5 +32
    return celsius.toString() + msg_1 + result.toString() + msg_2
}

function inchTocm(){
    var inches = prompt("Enter a number: ")
    var result = parseFloat(inches) * 2.54
    var message = inches + " inches are " + result + " cm"
    return message
}

function mensaje() {
    let Edad = document.getElementById("edadUsuario").value

if (Edad >=18) {
    alert("Eres mayor de edad")
    return
}
else {
    alert("Eres menor de edad")
    return
}
}

File: Multiconverter/multiconverter.py
start = input("Do you want to convert a value? yes/no ")

def exchange(third):
    euros = int(third)
    result = int(euros) *0.98
    if third == 1:
        msge = " euro is "
    else:
        msge=" euros are "
    return str(third) + msge + str(result) + " CHF"


def temp_conv(third):
    celsius = int(third)
    result = int(celsius) * 9/5 + 32
    if third == 1:
        msgc= " degree Celsius is "
    else :
        msgc= " degrees Celsius are "

    return str(third) + msgc + str(result) + " degrees Farenheit"

def inchTocm(third):
    inches = int(third)
    result = int((inches)) * 2.54
    if third == 1:
        msgi = " inche is "
    else:
        msgi = " inches are "
    return str(third) + msgi + str(result) + " cm"



if start == "yes":
    second = input("Which conversion? (euros/celsius/inches) ")
    third = int(input("Which value do you want to convert? "))
    if second == "euros":
        print(exchange(third))
    
    elif second == "celsius":
        print(temp_conv(third))
    
        
    elif second == "inches":
        print(inchTocm(third))
    
    else:
        print("I can't do this conversion")
else:
    quit()




